lib/modules/platform/bitbucket-server/index.ts

```diff
@@ -335,7 +334,10 @@ export async function getPrList(refreshCache?: boolean): Promise<Pr[]> {
     const searchParams: Record<string, string> = {
       state: 'ALL',
     };
-    if (!config.ignorePrAuthor && config.username !== undefined) {
+    if (
+      !GlobalConfig.get('ignorePrAuthor', false) &&
+      config.username !== undefined
+    ) {
```

Comment:
I don't think that these changes to use GlobalConfig are necessary. We only use this for _security_ reasons to prevent user config from overriding global config, but that can't happen here as we haven't read user credentials before initRepo().

Reply:
Based on these comments:

https://github.com/renovatebot/renovate/blob/a9a17787c57f06ab096cfef31fb4a3dfcb9812c2/lib/config/types.ts#L99-L101


https://github.com/renovatebot/renovate/blob/a9a17787c57f06ab096cfef31fb4a3dfcb9812c2/lib/config/types.ts#L131-L133

Reply:
Good point, although in reality it's "Config options used prior to reading in repo config" or not

Reply:
But I guess we can keep as-is so there's a clean separation